cpp/mcap/include/mcap/visibility.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#if defined _WIN32 || defined __CYGWIN__
+#ifdef __GNUC__
+#define MCAP_EXPORT __attribute__((dllexport))
+#define MCAP_IMPORT __attribute__((dllimport))
+#else
+#define MCAP_EXPORT __declspec(dllexport)
+#define MCAP_IMPORT __declspec(dllimport)
+#endif
+#ifdef MCAP_BUILDING_DLL
```

Comment:
Where does `MCAP_BUILDING_DLL` get set?

Reply:
I'll add a comment here in-file at least so it's not totally undocumented